### DISCUSSION TITLE
I can't get the same result as in the advanced example CO2 Plume Evolution and Leakage Through an Abandoned Well.

### DISCUSSION THREAD

**Discussion by ghost** [GEOS-DEV/GEOS#2658] [Q&A]
I have a problem with the CO2 Plume Evolution and Leakage Through an Abandoned Well example.

It took a little more than two days for my computer to reach cycle 500; I don't know if that is an expected behavior,  but I decided to stop GEOSX due to concern about the high temperature of my computer.

Then, I plotted the GEOS leakage rate using the source code, isothermalLeakyWellFigure.py, but unfortunately, as you can see in the figure below, it is utterly different. 

![lv_t](https://github.com/GEOS-DEV/GEOS/assets/125364509/f9741d7f-66b0-4c91-8462-a4278f8d2885)

My Geosx configuration is the following:
Num ranks: 1
Max threads: 20
GEOSX version: 0.2.0 (develop, sha1: 7d530bc92)
  - c++ compiler: gcc 12.2.0
  - openmp version: 201511
  - MPI version: Open MPI v4.1.4, package: Debian OpenMPI, ident: 4.1.4, repo rev: v4.1.4, May 26, 2022
  - HDF5 version: 1.12.1
  - Conduit version: 0.8.2
  - VTK version: 9.1.0
  - RAJA version: 2022.3.0
  - umpire version: 2022.3.0
  -  adiak version: ..
  - caliper version: 2.8.0
  - METIS version: 5.1.0
  - PARAMETIS version: 4.0.3
  - scotch version: 6.0.9
  - superlu_dist version: 6.3.0
  - suitesparse version: 5.7.9
  - Python3 version: 3.11.2
  - hypre release version: 2.28.0

Can you provide any insights into what might be causing the plot error?

**Comment by francoishamon** [comment]
Hello @Rosknub,

Regarding the run time, do you have access to a cluster? The mesh has about 678,720 cells, so my advice is to run the case with about 65 MPI ranks. This will accelerate the case very significantly. 

Regarding the validity of the results, could you please look at the results in Paraview first to make sure they make sense? This first "eye test" will be useful to figure out if something is wrong in the simulation, or if instead we should look at the hdf5 output used to generate the 2D plot.

Also, please share the full simulation log so we can see if something goes wrong during the simulation.

I ran this problem about 3 weeks ago and validated the results visually, but did not look at the 2D plot that you are showing.

**Reply by ghost** [reply]
Hello @francoishamon,

I had to research what a cluster is. I believe it's a group of connected Linux computers working together. Unfortunately, I only have one computer.

I apologize, but I am unfamiliar with how to run the case with 65 MPI ranks. Would a cluster be necessary for this task? 

I appreciate your response. Thank you very much.

**Comment by paveltomin** [comment]
reported here https://github.com/GEOS-DEV/GEOS/issues/2703
closing the discussion as duplicate
